### DISCUSSION TITLE
The IOMMU event define in opCode-85.txt

### DISCUSSION THREAD

**Discussion by LizYou** [opcm/pcm#321] [Q&A] [answered]
I found the PCM tool could trace the VT-d related events such as IOTLB hit and miss. And the events is defined in opCode-xx.txt file. 
However I couldn't find these events description in either intel SDM or VT-d spec. Does anybody know the document that contains  this event info?

**Comment by ilia-kurakin** [comment, marked as answer]
@LizYou , please refer to uncore performance monitoring guides ([1](https://www.intel.com/content/www/us/en/processors/xeon/scalable/xeon-scalable-uncore-performance-monitoring-manual.html), [2](https://software.intel.com/content/dam/develop/external/us/en/documents-tps/639778%20ICX%20UPG%20v1.pdf)) to find IIO event descriptions by event code (ev_sel) and unit mask (umask).
